source/server/server.cc

```diff
@@ -276,7 +276,7 @@ void InstanceImpl::updateServerStats() {
   server_stats_->total_connections_.set(listener_manager_->numConnections() +
                                         parent_stats.parent_connections_);
   server_stats_->days_until_first_cert_expiring_.set(
-      sslContextManager().daysUntilFirstCertExpires());
+      sslContextManager().daysUntilFirstCertExpires().value());
```

Comment:
Does this need to handle the case where there's an expire cert using `value_or`?

Reply:
It always return value:
https://github.com/envoyproxy/envoy/blob/4f8938c9f3c949da8c1c82c752a359c2555aaf71/source/server/ssl_context_manager.cc#L28